tests/clients/test_logging.py

```diff
@@ -3,74 +3,77 @@
 
 import logging
 import unittest
-from instana.singletons import agent, tracer
 
+from opentelemetry.trace import SpanKind
+
+from instana.singletons import agent, tracer
 
 class TestLogging(unittest.TestCase):
-    def setUp(self):
-        """ Clear all spans before a test run """
-        self.recorder = tracer.recorder
+
+    def setUp(self) -> None:
+        """Clear all spans before a test run"""
+        self.recorder = tracer.span_processor
         self.recorder.clear_spans()
-        self.logger = logging.getLogger('unit test')
+        self.logger = logging.getLogger("unit test")
 
-    def tearDown(self):
-        """ Ensure that allow_exit_as_root has the default value """
+    def tearDown(self) -> None:
+        """Ensure that allow_exit_as_root has the default value"""
         agent.options.allow_exit_as_root = False
 
-    def test_no_span(self):
-        with tracer.start_active_span('test'):
-            self.logger.info('info message')
+    def test_no_span(self) -> None:
+        with tracer.start_as_current_span("test"):
+            self.logger.info("info message")
 
         spans = self.recorder.queued_spans()
         self.assertEqual(1, len(spans))
```

Comment:
Let's use the [standard Python assert](https://docs.pytest.org/en/7.1.x/how-to/assert.html) expression from now on. 
Could you please update this test file to use it? Use always the pattern `assert value_to_evaluate <operation> expected_value`. 
```suggestion
        assert len(spans) == 1
```

Reply:
Done